pkg/reserved_resource/reserved_resource.go

```diff
@@ -0,0 +1,60 @@
+package reserved_resource
+
+import (
+	"fmt"
+	"time"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+)
+
+type ReservedResource struct {
+	AvailabilityZoneType      string
+	BillingMarketplaceAccount string `json:",omitempty"`
+	BillingModel              BillingModel
+	BYOC                      bool
+	Count                     int
+	CreatedAt                 time.Time
+	ResourceName              string
+	ResourceType              string
+	UpdatedAt                 time.Time
+}
+
+type BillingModel string
+
+const (
+	BillingModelMarketplace      BillingModel = "marketplace"
+	BillingModelMarketplaceAWS   BillingModel = "marketplace-aws"
+	BillingModelMarketplaceRHM   BillingModel = "marketplace-rhm"
+	BillingModelMarketplaceAzure BillingModel = "marketplace-azure"
+	BillingModelStandard         BillingModel = "standard"
```

Comment:
is it possible to use a string directly? This would avoid the need of updating this code whenever a new billing_model is added.